prime-finder.js

```diff
@@ -1,3 +1,27 @@
 module.exports = function(n) {
   // do work here
+  let count = 1;
+  let prime = 2;
+  let start = 3;
+
+  function findPrime(number) {
+    for (let x = 2; x < number; x++) {
```

Comment:
This will go faster if you don't check every number up to number but just up to the square root of number.